frontend/src/components/login/login.js

```diff
@@ -13,16 +18,32 @@ export class Login extends React.Component {
                 <div className="form">
                     <div className="form-group">
                         <label htmlFor="username">Username</label>
-                        <input type="text" name="username" placeholder="username"/>
+                        <input type="text" name="username" placeholder="username" onChange = {e => this.setState({username: e.target.value})}/>
                     </div>
                     <div className="form-group">
                         <label htmlFor="password">Password</label>
-                        <input type="password" name="password" placeholder="password"/>
+                        <input type="password" name="password" placeholder="password" onChange = {e => this.setState({pass: e.target.value})}/>
                     </div>
                 </div>
             </div>
             <div className="footer">
-                <button type="button" className="btn">
+                <button type="button" className="btn"
+                onClick={async () => {
+                    fetch('/auth/login', {
+                        method: 'POST',
+                        body: JSON.stringify({
+                          "username":this.state.username,"password":this.state.pass
+                        })
+                      } ).then(
+                        res => res.json()
+                        ).then(
+                          token => {
+                            console.log(this.state)
+                            this.setState({token: token});
+                            console.log(token)
```

Comment:
the console.log statements are good for debugging but I feel like they should be commented out when pushing to the main branch